packages/google-cloud-kms-inventory/test/gapic_key_dashboard_service_v1.ts

```diff
@@ -270,9 +270,14 @@ describe('v1.KeyDashboardServiceClient', () => {
         throw err;
       });
       assert(client.keyDashboardServiceStub);
-      client.close().then(() => {
-        done();
-      });
+      client
+        .close()
+        .then(() => {
+          done();
```

Comment:
I think this would better be placed in the finally block unless if you're handling the `done()` outside this